<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>PasswordBase</class>
 <widget class="QWidget" name="PasswordBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Passcode</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>6</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="prompt" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>1</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>Enter passcode</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="display" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maxLength" >
      <number>8</number>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item row="2" column="1" >
      <widget class="QPushButton" name="button_8" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="button_2" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QPushButton" name="button_1" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QPushButton" name="button_5" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QPushButton" name="button_4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="button_3" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QPushButton" name="button_0" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QPushButton" name="button_7" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="QPushButton" name="button_OK" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QPushButton" name="button_9" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QPushButton" name="button_6" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>6</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>button_1</tabstop>
  <tabstop>button_2</tabstop>
  <tabstop>button_3</tabstop>
  <tabstop>button_4</tabstop>
  <tabstop>button_5</tabstop>
  <tabstop>button_6</tabstop>
  <tabstop>button_7</tabstop>
  <tabstop>button_8</tabstop>
  <tabstop>button_9</tabstop>
  <tabstop>button_0</tabstop>
  <tabstop>button_OK</tabstop>
  <tabstop>display</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
